File: IronVault/IronVault.Model/Models/KorisnikUloga.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class KorisnikUloga
    {
        public int KorisnikUlogaId { get; set; }
        public int KorisnikId { get; set; }
        public int UlogaId { get; set; }
        public DateTime DatumIzmjene { get; set; }
        public virtual Uloga Uloga { get; set; } = null!;
    }
}

File: IronVault/UI/ironvault_desktop/lib/models/korisnik_uloga.dart
import 'package:ironvault_desktop/models/uloga.dart';
import 'package:json_annotation/json_annotation.dart';

part 'korisnik_uloga.g.dart';

@JsonSerializable()
class KorisnikUloga {
  int? korisnikUlogaId;
  int? korisnikId;
  int? ulogaId;
  Uloga? uloga;
  DateTime? datumIzmjene;

  KorisnikUloga({this.korisnikUlogaId});

  factory KorisnikUloga.fromJson(Map<String, dynamic> json) =>
      _$KorisnikUlogaFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$KorisnikUlogaToJson(this);
}
